# ARGquaculture
## Worldwide metagenomic monitoring of aquaculture-related antibiotic resistance
Aquaculture has been established as key to ensuring food security for millions of people, with a contribution of 45 % to fish production in 2018. To maintain and evolve aquaculture production despite pathogenic agents, antimicrobial agents have been used widely in the industry, as with any other terrestrial livestock production, which inevitably will lead to a rise in antimicrobial resistance (AMR). The global demand for fish and seafood is projected to increase by 30% by 2030, and aquaculture is expected to play a crucial role in meeting this demand, emphasising the importance of global aquaculture and indicating a market and production under meteoric rise with an annual growth of 5.8 % since 2001 (FAO 2021).  It is expected that the uncontrolled use of antimicrobials in aquaculture farms will cause a rise (Romalde and Toranzo 2002; Vendrell et al. 2006; Chen et al. 2020; Preena et al. 2020).

Despite this rise in AMR, worldwide agencies are lacking actual way to monitor and survey the extend of AMR related to Aquaculture (FAO 2021). Such surveillance is critical to understand the extend of the AMR problem.

Here we present the metagenomic workflow for mining metagenomic data for antibiotic resistance genes (ARGs) in metagenomic data derived from Aquaculture. We generate metagenomic assembled genomes (MAGs) and viral draft genomes (vOTUs) to investage carriers of ARGs in local environments and generate a risk assesment of ARGs.

Subsequent analysis relates ARGs, MAGs, and vOTUs to known compounds used in Aqaculture for a more causal relationship. 

All code for this investigation are available in this repository. 

